back/app/controllers/web_api/v1/custom_field_bins_controller.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+class WebApi::V1::CustomFieldBinsController < ApplicationController
+  before_action :set_custom_field, only: [:index]
+  before_action :set_custom_field_bin, only: [:show]
+  skip_after_action :verify_policy_scoped
+
+  def index
+    @bins = @custom_field.custom_field_bins
+    @bins = @bins.order(:created_at)
+    @bins = paginate(@bins)
+    render json: linked_json(@bins, WebApi::V1::CustomFieldBinSerializer, params: jsonapi_serializer_params)
```

Comment:
Does `@bins` need to be a global variable?

Reply:
It doesn't, changing